### PR TITLE
Bump all examples to version to 0.48.1

### PR DESCRIPTION
**Pull request by timtwang**
Bump all examples to version 0.48.1 for public preview

### CODE REVIEW

**Review by tylerbutler — APPROVED** [review]

**Review by sumedhb1995 — APPROVED** [review]
:shipit:

File: brainstorm/package.json [hunk 1]
```diff
@@ -45,9 +45,9 @@
   },
```
Comment (sumedhb1995) [inline brainstorm/package.json:45]: Could we also bump the versions of the examples themselves at the top of package.json to 0.48.1 as well to match the packages?

Comment (timtwang) [reply brainstorm/package.json:45]: updated

**Review by sumedhb1995 — APPROVED** [review]

**Review by tylerbutler — APPROVED** [review]
🖖🏻

**Pull request merged**